pytorch_toolkit/formula_recognition/README.md

```diff
@@ -81,6 +81,36 @@ In addition to common parameters you can specify the following arguments:
 - `train_transforms_list` - similiar to `val_transforms_list`
 - `epochs` - number of epochs to train
 
+One can use some pretrained models. Right now two models are available:
+* medium model:
+    * [checkpoint link](https://download.01.org/opencv/openvino_training_extensions/models/formula_recognition/medium_photograped_0185.pth)
+    * digits, letters, some greek letters, fractions, trigonometric operations are supported; for more details, please, look at corresponding vocab file
+    * to use this model, please, change these fields in the config file:
+    ```
+    backbone_config:
+        arch: resnext50_32x4d
+        disable_layer_3: true
+        disable_layer_4: true
+    model_path: <path to the model>
+    vocab_path: vocabs/vocab_medium.json
+    ```
```

Comment:
Why don't you just put the ready to use config file in the repo?..

Reply:
Updated config to use medium photographed model by default